harness/determined/common/api/bindings.py

```diff
@@ -478,22 +478,30 @@ def __init__(
         self,
         *,
         agentGid: "typing.Optional[int]" = None,
+        agentGroup: "typing.Optional[str]" = None,
```

Comment:
should we switch the CLI set agent user group over with this change? 

Waiting to do it at all once or bit by bit as we make sure functionality is matched both seem fine to me but curious on your thoughts.

Reply:
this is a judgement call: on one hand, you are right in advocating this. on the other, I am working from a mindset of a critical path towards storage solution, and we basically have this work  outlined and ticketed as one of our four top tech debt items.